commit 8832fe249ae46bed3b4d0c4f39604435c019627d
Author: Josh Waihi <josh.wahi@acquia.com>
Date:   Fri Mar 6 10:19:51 2015 +1300

    Updated webform

diff --git a/drupal-org.make b/drupal-org.make
index 43e223a..ee9f82a 100644
--- a/drupal-org.make
+++ b/drupal-org.make
@@ -120,7 +120,7 @@ projects[view_unpublished][download][revision] = e9df1d3f33b93410bb62e65f7098128
 projects[view_unpublished][download][type] = "git"
 projects[view_unpublished][download][url] = "http://git.drupal.org/project/view_unpublished.git"
 projects[view_unpublished][type] = "module"
-projects[webform][version] = "4.1"
+projects[webform][version] = "4.5"
 projects[webform_clear][version] = "2.0"
 projects[webform_phone][version] = "1.19"
 projects[workbench][version] = "1.2"
