From abcbb15a0943c8a337f9a3aed5552da792311588 Mon Sep 17 00:00:00 2001
From: Kim Pepper <kim@pepper.id.au>
Date: Thu, 26 Feb 2015 10:47:12 +1100
Subject: [PATCH] 29717 Update ctools to 7.x-1.7

---
 drupal-org.make | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drupal-org.make b/drupal-org.make
index b74c31a..129fdb1 100644
--- a/drupal-org.make
+++ b/drupal-org.make
@@ -18,7 +18,7 @@ projects[chosen][patch][] = https://drupal.org/files/2021639-pin-chosen-26.patch
 projects[chosen][version] = 2.0-alpha2
 projects[context][version] = "3.6"
 projects[crumbs][version] = "2.0-beta19"
-projects[ctools][version] = "1.4"
+projects[ctools][version] = "1.7"
 projects[date][version] = "2.8"
 projects[defaultconfig][patch][] = "https://drupal.org/files/1940366-1-illegal-offset-warning.patch"
 projects[defaultconfig][version] = "1.0-alpha9"
