diff --git a/drupal-org.make b/drupal-org.make
index b3237c8..33b72e0 100644
--- a/drupal-org.make
+++ b/drupal-org.make
@@ -73,7 +73,7 @@ projects[node_clone][version] = "1.0-rc2"
 projects[oauth][version] = "3.2"
 projects[panels][version] = "3.4"
 projects[paranoia][version] = "1.3"
-projects[password_policy][version] = "1.9"
+projects[password_policy][version] = "1.11"
 projects[pathauto][version] = "1.2"
 projects[pathauto_persist][patch][] = "https://www.drupal.org/files/issues/pathauto_persist-n2107365-7.patch"
 projects[pathauto_persist][version] = "1.x-dev"
